controllers/defaults.go

```diff
@@ -5,5 +5,5 @@ package controllers
 // ***********************
 
 const (
-	InstaScaleImage = "quay.io/project-codeflare/instascale-controller:v0.0.3"
+	InstaScaleImage = "quay.io/project-codeflare/instascale-controller:v0.0.4"
```

Comment:
@astefanutti if we don't edit this file how else would we update the default?

Reply:
It's now to be set in the Makefile: https://github.com/project-codeflare/codeflare-operator/blob/a71303b7d9a2a59eb8310dbbaa351e2b43af256c/Makefile#L13

Ideally, we'll eventually have the dependencies defined there in a central place.

Reply:
And run `make defaults`, but it's done with most make targets.

Reply:
Got it!

Reply:
Fixed.